public/data/nazmulhassan.json

```
@@ -0,0 +1,27 @@
+{
+    "name": "MD NAZMUL HASSAN",
+    "type": "personal",
+    "bio": "Open Source Enthusiast!",
+    "avatar": "https://github.com/nazmul19.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/nazmul19",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/nazmulhassan19",
+        "icon": "twitter"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started Sharing and Learning",
+        "date": "December 2021",
+        "icon": "Globe",

```

```suggestion
    "icon": "globe",
```
It doesn't matter the case of the icon, but because all the other icons are in lowercase, to format the code, I suggest you make this lowercase too.